packages/stark-core/src/logging/services/logging.service.spec.ts

```diff
@@ -181,7 +182,34 @@ describe("Service: StarkLoggingService", () => {
 			expect((<StarkLogMessage>message).message).toContain("end of message");
 			expect((<StarkLogMessage>message).timestamp).toBeDefined();
 			expect((<StarkLogMessage>message).error).toBeDefined();
-			expect((<StarkLogMessage>message).error).toContain("this is the error");
+
+			const error: StarkError = (<StarkError>(<StarkLogMessage>message).error);
+			if (error) {
```

Comment:
I would suggest to replace this if block by this:

```javascript
expect(error).toBeDefined();  // so the test will fail in case it is undefined
expect((<string>(error).message)).toContain("this is the error");
```